docs/modules/ROOT/pages/how-tos/compile-catalog.adoc

```diff
@@ -0,0 +1,76 @@
+= Compile a catalog
+
+.Goal
+Compile and push a cluster catalog from local machine to update dependencies.
+
+== Requirements
+
+* A https://syn.tools/commodore/running-commodore.html[working commodore command].
+* An access token to a running Lieutenant instance. Storing the token as a kubeconfig is recommended.
+* An existing Cluster in Lieutenant.
+
+== Prepare Commodore
+
+The following snippet sets up needed environment variables in your current working directory.
+
+TIP: You can skip this step if you have already defined them already with another method.
+
+[source,bash]
+----
+# Assuming "syn-synfra" is the user in your kubeconfig
+LIEUTENANT_TOKEN=$(kubectl config view -o jsonpath='{.users[?(@.name == "syn-synfra")].user.token}'  --raw)
+
+echo LIEUTENANT_TOKEN=\""${LIEUTENANT_TOKEN}"\" >> .env
+echo LIEUTENANT_AUTH=\""Authorization:Bearer ${LIEUTENANT_TOKEN}"\" >> .env
+echo LIEUTENANT_URL="the-public-lieutenant-API-URL" >> .env
+echo LIEUTENANT_NS="lieutenant-instance" >> .env
+echo COMMODORE_API_TOKEN=\""${LIEUTENANT_TOKEN}"\" >> .env
+echo COMMODORE_API_URL="${LIEUTENANT_URL}" >> .env
+
+# Double check the environment variables
+less .env
+
+# Export the variables
+set -a; source .env; set +a
+----
+
+== Compilation
+
+.Choose cluster
+. Get the Cluster ID from Lieutenant.
++
+[source,bash]
+----
+kubectl get cluster -n <lieutenant-instance>
```

Comment:
Alternatively `commodore catalog list -v`

Reply:
They're very similar indeed. I could update the change-parameter guide so it builds on top of compile-catalog.

Reply:
I just compiled without LIEUTENANT_NS and it still worked.